### PR TITLE
Translate new strings

### PR DESCRIPTION
**Pull request by frithnanth**
- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Screenshots added (for bigger UI changes)
- [ ] Manually tested changed features in running JabRef
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)
- [ ] If you changed the localization: Did you run `gradle localizationUpdate`?

### CODE REVIEW

File: src/main/resources/l10n/JabRef_it.properties [hunk 1]
```diff
@@ -2275,13 +2275,13 @@ last_four_nonpunctuation_characters_should_be_numerals=gli_ultimi_quattro_caratt
...
-get_involved=
-Used_libraries=
-MIT=
+Tools=Strumenti
+What\'s_new_in_this_version?=Cosa_c\'è_di_nuovo_in_questa_versione?
+Want_to_help?=Ti_serve_aiuto?
+Make_a_donation=Fai_una_donazione
+get_involved=collabora_con_noi
+Used_libraries=Librerie_usate
+MIT=MIT
```
Comment (stefan-kolb) [inline src/main/resources/l10n/JabRef_it.properties]: @JabRef/developers Why do we have a translation for MIT?

Comment (Siedlerchr) [inline src/main/resources/l10n/JabRef_it.properties]: My first guess it is used somewhere in the About dialog. The license. But makes no sense to have it in the localization

Comment (stefan-kolb) [inline src/main/resources/l10n/JabRef_it.properties]: Yeah, we should fix this when merging this PR!

Comment (Siedlerchr) [inline src/main/resources/l10n/JabRef_it.properties]: <Hyperlink onAction="#openLicense" text="%MIT" /> in the AboutDialog.fxml

**Comment by Siedlerchr** [comment]
I merge this for the moment and will fix this then afterwards in all lang files

**Pull request merged**